### ISSUE TITLE
Plan designs in the Custom Drafts Plan Designs folder have a plan type of “custom” but the HR plan design type still shows as “standard”

### ISSUE THREAD

**Issue by g4gaurang** [willdwyer/bcbsmaissuestracker#242]
From: Goradia, Gaurang [mailto:Gaurang.Goradia@bcbsma.com] 
Sent: Friday, July 31, 2015 11:00 AM
To: Michelle Parrish
Subject: Plan designs in the Custom Drafts Plan Designs folder have a plan type of “custom” but the HR plan design type still shows as “standard”.

Hi Michelle,

Why is it that all plan designs in the Custom Drafts Plan Designs folder have a plan type of “custom” but the HR plan design type still shows as “standard”.
When system creates a copy of the standard plan design to create a custom plan why is the HR plan design type not changed to Custom ?
It is misleading because when the user does a search on all plan design types of “standard” even these custom plan designs show up in the results.
The plan design type should also be changed to “Custom” during the creation of a custom plan from a standard plan. 

Thanks,
Gaurang

![image](https://cloud.githubusercontent.com/assets/10272072/9232018/54799bb6-40f9-11e5-9a74-e1f0e034c09a.png)

---

From: Steve Hollingsworth [mailto:shollingsworth@highroads.com] 
Sent: Tuesday, August 04, 2015 4:13 PM
To: Goradia, Gaurang; Michelle Parrish
Cc: Dwyer, William; DeCoste, Jennifer
Subject: RE: Plan designs in the Custom Drafts Plan Designs folder have a plan type of “custom” but the HR plan design type still shows as “standard”.

Hi Gaurang,
The Plan Type attribute is a system required attribute utilized by the plan configurator, whereas Plan Design Type is a User Defined Attribute (UDA) which is independent of the configurator.
Since Plan Type is tied in directly with plan configurator, the system knows to flip it from standard to custom once the user makes the selection and saves. All UDA values are inherited from the base plan that the custom is getting created from, so this is why you are seeing the difference.
Let me know if this doesn’t make sense, or if you have any other questions.
Thanks!
Steve

---

From: Goradia, Gaurang [mailto:Gaurang.Goradia@bcbsma.com] 
Sent: Wednesday, August 05, 2015 3:01 PM
To: Steve Hollingsworth; Michelle Parrish
Cc: BCBS.William.Dwyer; DeCoste, Jennifer
Subject: RE: Plan designs in the Custom Drafts Plan Designs folder have a plan type of “custom” but the HR plan design type still shows as “standard”.

Hi Steve,

Was the plan type attribute added specifically for the plan configurator ?
Is it possible to flip the value of the Plan Design Type dynamically from standard to custom when system creates custom from a standard and assigns a plan type of custom ?
If not, it will add an additional step in our attributes assignment process of flipping back the plan design type from standard to custom for all custom files that we create through the system. - If we do not do that, any searches/reports  we do using the plan design type attribute will be inaccurate. We use the plan design type attribute to identify a standard v/s custom plan. There are currently 15,410 files in the system with a the plan design type assigned.

Thanks,
Gaurang

---

Hi Gaurang,
You are correct, the plan type attribute is a system attribute required by the plan configurator. 

As previously mentioned, since it’s a UDA, plan design type has to be set/updated by the user and cannot be dynamically updated as you describe.

Thanks
Steve H

**Comment by michparrish** [comment]
BCBS and HR met this morning to go over a potential solution to making this field readily available to the user to update - add field into the Plan Details tab of the Configurator and allow the user to update as they are making their changes to the Custom plan. However, it's understood that BCBS does not want to add another step to the process for the users. Also, being that this is a UDF and is already filled in for the Standard plan with the value of 'Standard', any 'mandatory' rule on this field would be moot.

BCBS decided that Will will perform the daily task of searching for plan designs created the prior day and output plan details/bulk edit the Plan Design Type attribute and upload. Will will meet with Abbie to run this process by her for approval and update this item.

**Comment by g4gaurang** [comment]
Will, please work with Abbie and once approved, document  and set up the periodic process for execution. Once you have things set up, you can document and close this bug.

**Comment by markderg** [comment]
No action for HR.  BCBSMA will work on developing internal process and document next steps.

**Comment by willdwyer** [comment]
Hello,

I discussed this process with Abbie today, and it has been approved.  Going forward, I will perform the daily task of searching for plan designs created the prior day and bulk edit Plan Design Type attributes.  This isn't a very time consuming process, so it is fine with me.

I have made a note to add this to Operations Manual, along with Account Name/Account Number matching, as well as monitoring WIP items.

This request is now being closed.
